scripts/verify-release.sh

```diff
@@ -5,4 +5,6 @@ echo "====================================================="
 echo "= Running automated tests before publishing release ="
 echo "====================================================="
 set -x
-./gradlew clean buildCodeSearchAssets buildPluginAndAssertAgentBinariesExist runPluginVerifier
+unset CODY_DIR
+./gradlew clean
```

Comment:
was there any trouble without it?

Reply:
Gradle was failing when I included `clean` with the other tasks. I don't really understand Gradle, it seems like a hot mess compared to other build tools I've used.